CactbotOverlay/CactbotEventSource.cs

```diff
@@ -83,11 +84,19 @@ public class CactbotEventSource : EventSourceBase, ILogger {
 
     public delegate void PartyWipeHandler(JSEvents.PartyWipeEvent e);
     public event PartyWipeHandler OnPartyWipe;
+
+    public delegate void FateEventHandler(JSEvents.FateEvent e);
+    public event FateEventHandler OnFateEvent;
+
     public void Wipe() {
       Advanced_Combat_Tracker.ActGlobals.oFormActMain.EndCombat(false);
       OnPartyWipe(new JSEvents.PartyWipeEvent());
     }
 
+    public void DoFateEvent(JSEvents.FateEvent e) {
```

Comment:
Maybe calls to DoFateEvent should just go to OnFateEvent and this function shouldn't exist?

Reply:
You can't raise events from other classes in C#. Calling `OnFateEvent` from `FateWatcher` wouldn't work.
However, you could just scrap the `OnFateEvent` and call `DispatchToJS` from `FateWatcher`.

Reply:
Done.